package.json

```diff
@@ -31,7 +31,7 @@
     "@types/jest": "^24.0.13",
     "codecov": "^3.5.0",
     "commitlint": "^8.0.0",
-    "core-js": "^3.1.4",
+    "core-js": "^3.2.1",
```

Comment:
Can you tell why we need these changes?

Reply:
@osdnk jest was giving some issue. this is only for tests so not very important.